engine/render/src/render/render_private.h

```diff
@@ -255,6 +255,10 @@ namespace dmRender
 
     Result GenerateKey(HRenderContext render_context, const Matrix4& view_matrix);
 
+    // JG: Should these be a dmsdk function?
+    void SetNamedConstant(HNamedConstantBuffer buffer, dmhash_t name_hash, dmVMath::Vector4* values, uint32_t num_values, dmRenderDDF::MaterialDesc::ConstantType constant_type);
+    bool GetNamedConstant(HNamedConstantBuffer buffer, dmhash_t name_hash, dmVMath::Vector4** values, uint32_t* num_values, dmRenderDDF::MaterialDesc::ConstantType* constant_type);
+
```

Comment:
These are the same functions as in the sdk but with the extra type argument added. They are not used anywhere externally so I guess we can wait with that until it's needed/requested?

Reply:
Had to move these to DMSDK for particlefx